ops/ansible/roles/bfd-server/files/bluebutton-appserver-config.sh

```diff
@@ -1,336 +0,0 @@
-#!/bin/bash
-
```

Comment:
Very happy to see this file go. Failures within this script were not fun to debug.

Reply:
Same!

Hated the thing from the moment I wrote it.